.mockery.yaml

```diff
@@ -1,14 +1,14 @@
-with-expecter: false
+with-expecter: true
 disable-version-string: true
 dir: internal/testutil/mocksvc
 outpkg: mocksvc
 filename: "{{ .InterfaceName | snakecase }}.go"
 mockname: "{{.InterfaceName}}"
 
 packages:
-  github.com/mongodb/terraform-provider-mongodbatlas/internal/service/searchdeployment:
+  go.mongodb.org/atlas-sdk/v20231115007/admin:
```

Comment:
i'm curious to see if the SDK upgrade will update this

Reply:
if not, we could add an additional step in the makefile to include this.

Reply:
confirmed gomajor does not modify the mockery file, added an additional step in our update-sdk script to handle this (will port to CFN as well)

Reply:
nice